.eslintrc.yaml

```diff
@@ -65,10 +66,13 @@ rules:
   quotes: [2, single, avoid-escape]
   semi: [2, always]
   semi-spacing: [2, {before: false, after: true}]
+  sort-keys: [2, asc, {caseSensitive: true, natural: true}]
```

Comment:
let's do `caseSensitive: false` (even though I doubt that we will have a mix case in any list.. but it is still better I guess.